### PR TITLE
rename the policy file to container_device

### PR DESCRIPTION
**Pull request by mregmi**
rename the policy file to container_device
Signed-off-by: Manish Regmi <manish.regmi@intel.com>

### CODE REVIEW

**Comment by Walnux** [comment]
Could you split this PR into two PRs. one is for updating the readme, and another is for changing the policy file name.

File: README.md [hunk 1]
```diff
@@ -1,24 +1,38 @@
...
+This repository contains user tailored container SELinux policies and the method for deploying these policies on Red Hat OpenShift Container Platform. These SELinux policies are required for the device plugins and the workloads leveraging these device plugins in [intel-device-plugins-for-kubernetes](https://github.com/intel/intel-device-plugins-for-kubernetes) to deploy on Red Hat OpenShift Container Platform with proper SELinux permissions.
 
-These policies create new domains called `container_device_t` for user workloads, `container_device_plugin_t` for device plugins and `container_device_plugin_init_t` for init containers. The device plugins run as that label to have all the necessary privileges.
-These policies are derived from the [container-selinux](https://github.com/containers/container-selinux) project.
+These policies create new domains called `container_device_t` for user workloads, `container_device_plugin_t` for device plugins and `container_device_plugin_init_t` for init containers. The device plugins are labeled as container_device_plugin_t by selinux so that it can access proper resources on the host.
```
Comment (Walnux) [inline README.md:4]: so it can be assigned the proper permissions to access the resources on the host.

File: README.md [hunk 2]
```diff
@@ -1,24 +1,38 @@
...
+These policies are derived from the corresponding policies in [container-selinux](https://github.com/containers/container-selinux) project.
 
-## Building the SELinux policies (for developer)
+## Building and Deploying the SELinux policies in RHEL
 
-The policies are part of the container image and built using `docker build`. To build the policies, install the selinux-policy development package using below commands: 
+We have only verified these on `RHEL` and `centos`. To build the policies, please follow the steps below:
```
Comment (Walnux) [inline README.md:9]: Please add RHEL version and centos version.

File: README.md [hunk 3]
```diff
@@ -1,24 +1,38 @@
...
 
-These policies create new domains called `container_device_t` for user workloads, `container_device_plugin_t` for device plugins and `container_device_plugin_init_t` for init containers. The device plugins run as that label to have all the necessary privileges.
-These policies are derived from the [container-selinux](https://github.com/containers/container-selinux) project.
+These policies create new domains called `container_device_t` for user workloads, `container_device_plugin_t` for device plugins and `container_device_plugin_init_t` for init containers. The device plugins are labeled as container_device_plugin_t by selinux so that it can access proper resources on the host.
+These policies are derived from the corresponding policies in [container-selinux](https://github.com/containers/container-selinux) project.
 
-## Building the SELinux policies (for developer)
+## Building and Deploying the SELinux policies in RHEL
```
Comment (Walnux) [inline README.md:7]: Building and installing the SElinux policies

File: README.md [hunk 4]
```diff
@@ -1,24 +1,38 @@
...
 
-The policies are part of the container image and built using `docker build`. To build the policies, install the selinux-policy development package using below commands: 
+We have only verified these on `RHEL` and `centos`. To build the policies, please follow the steps below: 
 ```
-$ sudo dnf -y selinux-policy-devel
+$ sudo dnf -y install make selinux-policy selinux-policy-devel container-selinux
 $ make -f /usr/share/selinux/devel/Makefile
 ```
+The above commands will create a policy binary named `container_device.pp`. To deploy the policy, first copy it into the worker nodes of your cluster. then run:
```
Comment (Walnux) [inline README.md:14]: The above command creates a policy binary file named `container_device.pp`. To install the policy run:

File: README.md (hunk 2 above)
Comment (Walnux) [reply README.md:9]: add
Get source code 
```
$ export USER_CONTAINER_SELINUX_SRC=/path/to/user-container-selinux
$ cd $USER_CONTAINER_SELINUX_SRC
```

File: README.md [hunk 5]
```diff
@@ -1,24 +1,38 @@
...
+```
+$ sudo semodule -i container_device.pp
+```
+To verify that it is installed proper run the command below and verify that it lists `container_device`.
+```
+$ sudo semodule -l | grep container_device 
+```
```
Comment (Walnux) [inline README.md:21]: The container_device policy shows that the policy is properly installed.

File: README.md [hunk 6]
```diff
@@ -1,24 +1,38 @@
...
 
-## Deploying the SELinux policy (for end user)
+## Deploying the SELinux policy in openshift cluster
```
Comment (Walnux) [inline README.md:23]: on a Red Hat® OpenShift® cluster

File: README.md [hunk 7]
```diff
@@ -1,24 +1,38 @@
...
 
-To deploy the policies on OpenShift Container Platform, run: 
+Before deploying, make sure you have `oc` command installed and your openshift cluster is up and running. The instructions for downloading `oc` command can be found [here]{https://docs.openshift.com/container-platform/4.10/cli_reference/openshift_cli/getting-started-cli.html}. This policy is tested only for Openshift version 4.10. To deploy the policies on OpenShift Container Platform, run:
```
Comment (Walnux) [inline README.md:25]: Before deploying, make sure you have `oc` command installed and your work machine.
And your Cluster is up and running. 
This policy is tested on Red Hat® OpenShift® version 4.10

File: README.md [hunk 8]
```diff
@@ -1,24 +1,38 @@
...
+$ sudo semodule -l | grep container_device 
+```
 
-## Deploying the SELinux policy (for end user)
+## Deploying the SELinux policy in openshift cluster
 
-To deploy the policies on OpenShift Container Platform, run: 
+Before deploying, make sure you have `oc` command installed and your openshift cluster is up and running. The instructions for downloading `oc` command can be found [here]{https://docs.openshift.com/container-platform/4.10/cli_reference/openshift_cli/getting-started-cli.html}. This policy is tested only for Openshift version 4.10. To deploy the policies on OpenShift Container Platform, run: 
 ```
+$ oc login (if you are not already logged in)
```
Comment (Walnux) [inline README.md:27]: if you are not logged in

File: README.md [hunk 9]
```diff
@@ -1,24 +1,38 @@
...
 $ oc apply -f https://raw.githubusercontent.com/intel/user-container-selinux/main/policy-deployment.yaml
 ```
-
+To verify that it is installed proper run the command below and verify that it lists `container_device`.
```
Comment (Walnux) [inline README.md:30]: Notes: change the node to the node name you want to check the SELinux policy statues.

**Review by Walnux — CHANGES_REQUESTED** [review]

**Comment by mregmi** [comment]
as suggested i split the readme changes to a separate PR.

**Comment by Walnux** [comment]
Please split the PR into multiple PRs. each PR should on do one thing.

**Review by Walnux — APPROVED** [review]

**Pull request merged**